usagov/vote-gov-drupal
# VOTE-2341: Stage page addition fields

## Jira ticket

[VOTE-2341](https://cm-jira.usa.gov/browse/VOTE-2341)
## Description

Adds fields for state page addition for nvrf link/content

## Deployment and testing

### QA/Testing instructions

1. lando retune
2. add placeholder content to registration form selector
3. confirm content shows above nvrf registration dropdown

## Checklist for the Developer




## Checklist for the Peer Reviewers
